source/extensions/transport_sockets/tls/cert_validator/default_validator.cc

```diff
@@ -41,9 +41,9 @@ namespace TransportSockets {
 namespace Tls {
 
 DefaultCertValidator::DefaultCertValidator(
-    const Envoy::Ssl::CertificateValidationContextConfig* config, SslStats& stats,
-    TimeSource& time_source)
-    : config_(config), stats_(stats), time_source_(time_source) {
+    const std::string cert_name, const Envoy::Ssl::CertificateValidationContextConfig* config,
+    SslStats& stats, TimeSource& time_source)
+    : config_(config), stats_(stats), time_source_(time_source), cert_name_(cert_name) {
```

Comment:
probably modify the data member ordering which matches the constructor's argumetns